shared/actions/kbfs/index.desktop.js

```diff
@@ -5,7 +5,17 @@ import * as Constants from '../../constants/config'
 import {config} from '../../constants/types/keybase-v1'
 import type {AsyncAction} from '../../constants/types/flux'
 
-export function openInKBFS (path: string = ''): AsyncAction {
+const open = (kbfsPath: string, path: ?string) => {
+  shell.openItem(`${kbfsPath}${path}`)
+}
+
+// Paths MUST start with /keybase/
+export function openInKBFS (path: string = Constants.defaultKBFSPath): AsyncAction {
```

Comment:
let's treat paths in the app as /keybase always, we can swap out for windows specific stuff here